index.js

```diff
@@ -1 +1,22 @@
-// phew… not a lot going on here. Please add some code!
+const bookMarkToggle = document.querySelector('[data-js="bookMark"]');
+
+bookMarkToggle.addEventListener("click", () => {
+bookMarkToggle.classList.toggle("bookmark--active");
+});
+
+const card =document.querySelector('[class="card-list__item"]');
+
+const answerButton =document.querySelector('[class="card__button-answer"]');
```

Comment:
```suggestion
const showAnswerButton =document.querySelector('[class="card__button-answer"]');
```
This would be a bit more descriptive, or what do you think?